Makefile

```diff
@@ -30,7 +30,7 @@ APP_LOAD_PARAMS += --path "1517992542'/1101353413'"
 
 APPVERSION_M=1
 APPVERSION_N=9
-APPVERSION_P=11
+APPVERSION_P=14-shyft
```

Comment:
This change impacts the version of the Ethereum app. There is no reason for that.